AndroidUI/Utils/Screenshoter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AndroidUI.Driver;
```

Comment:
Maybe we can create ScreenshotBase.cs which will have common logic and then inherit this class and class in WebUI with overriding of functions?